### PR TITLE
Page/acq 708  code.org educate csd into a scroller

### PR DESCRIPTION
**Pull request by dmantonyuk**
Turn the unit and collections sections of code.org/educate/csd into a scroller

## Links
https://codedotorg.atlassian.net/browse/ACQ-708
https://code.org/educate/csd

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
image

File: pegasus/sites.v3/code.org/public/educate/csd.haml [hunk 1]
```diff
@@ -83,132 +83,16 @@ theme: responsive_full_width
...
-        .content-wrapper
-          %p.overline=hoc_s(:module_duration_month)
-          %h3=hoc_s(:curriculum_name_ai)
-          %img{src: "/images/ai/ai-curriclum-machine-learning.png", alt: ""}
-          %p=hoc_s(:ai_curriculum_desc_01)
-        .content-footer
-          %a.link-button{href: CDO.studio_url("/s/aiml-2023?viewAs=Instructor"), aria:{label: hoc_s(:ai_curriculum_btn_aria_label_01)}}
-            =hoc_s(:call_to_action_explore_unit)
 
-= view :section_divider_line
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/csd.haml:151]: Can you add this back so there's a divider line between the sections?

**Comment by dmantonyuk** [comment]
@kelbyhawn  please check again

File: pegasus/sites.v3/code.org/public/educate/csd.haml [hunk 2]
```diff
@@ -230,3 +116,58 @@ theme: responsive_full_width
     = view :support_and_questions_section
 
 = view :analytics_event_log_helper, event_name: AnalyticsConstants::CSD_CURRICULUM_PAGE_VISITED_EVENT
+
+
+%script{:src=>'/js/jquery.carouFredSel-6.2.1-packed.js'}
+%script{:src=>'/js/jquery.touchSwipe.min.js'}
+
+
+:javascript
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/csd.haml]: @dmantonyuk Javascript files can be added [here](pegasus/sites.v3/code.org/public/js) so they can be linted, and then just needs a script tag here.

File: pegasus/sites.v3/code.org/public/css/page/csd-styles.scss [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    right: -100vh;
+  }
+
+  &__prev {
+    position: absolute;
+    left: -100vw;
+  }
+
+  &--flexible_units {
+
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/page/csd-styles.scss]: Can this be removed?

File: pegasus/sites.v3/code.org/public/css/page/csd-styles.scss [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+      align-items: center;
+
+      .action-block {
+        width: 48%;
+        margin: 0;
+      }
+
+    }
+
+
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/page/csd-styles.scss]: Nits: extra whitespace here and on line 122

**Review by kelbyhawn — COMMENTED** [review]
Looks great! I left a couple comments/whitespace nits and sent a screenshot re: the action blocks using the Barlow font instead of Gotham, but once these are updated I'll take another look and it should be good to go 😄

**Comment by dmantonyuk** [comment]
download

File: pegasus/sites.v3/code.org/public/css/page/csd-styles.scss [hunk 5]
```diff
@@ -73,6 +73,11 @@
 
 .carousel-wrapper {
   padding-bottom: 0;
+
+  .carousel-wrapper p {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/page/csd-styles.scss]: The paragraphs are still showing the wrong font — this `.carousel-wrapper` class can be removed and then it should work 🙌

**Comment by dmantonyuk** [comment]
@kelbyhawn  please take a look

**Review by kelbyhawn — APPROVED** [review]
Looks great, thank you! 🎉

**Pull request merged**